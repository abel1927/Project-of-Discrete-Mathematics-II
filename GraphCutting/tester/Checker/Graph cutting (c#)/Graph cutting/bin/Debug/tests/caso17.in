80 150
9 69
14 67
60 36
48 55
5 50
69 40
8 19
63 35
73 36
63 29
65 53
14 63
60 59
38 61
41 26
19 57
11 63
9 34
52 12
63 18
13 75
46 78
3 1
64 18
71 7
19 28
70 3
9 16
61 42
45 61
32 79
73 41
3 77
36 61
29 36
22 48
71 23
12 4
20 9
44 59
45 53
68 47
65 68
47 42
47 28
77 10
3 31
78 44
69 49
54 57
5 28
11 51
11 13
64 41
18 46
7 64
39 69
35 34
51 61
45 20
51 13
11 50
29 13
42 28
63 30
26 49
12 26
12 53
29 54
57 21
20 75
1 46
3 79
22 76
38 57
33 20
36 75
6 46
75 44
14 51
28 17
38 17
75 58
26 36
36 44
78 13
16 73
19 56
74 60
19 45
51 27
11 79
11 62
53 40
47 17
75 68
2 28
18 22
8 3
23 6
56 51
31 1
44 62
5 16
24 50
71 78
35 3
5 3
48 61
64 48
2 78
6 44
77 60
1 43
80 48
5 55
6 30
50 62
55 68
69 45
61 16
31 38
24 35
58 31
24 17
47 3
78 42
76 10
41 5
71 29
56 41
3 6
12 1
13 80
57 11
65 78
52 77
53 60
60 61
55 24
42 49
53 24
15 53
73 29
40 25
35 69
76 3
72 34
68 52
39 2
