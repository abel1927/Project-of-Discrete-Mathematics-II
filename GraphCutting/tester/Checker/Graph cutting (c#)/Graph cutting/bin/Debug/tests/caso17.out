29 54 57
46 1 43
45 61 51
69 45 53
35 69 49
25 40 69
24 53 40
65 53 15
65 68 75
78 71 29
42 78 65
75 44 78
31 58 75
61 38 31
60 77 52
61 60 74
44 59 60
11 62 44
24 50 62
68 55 24
52 68 47
53 12 52
36 60 53
41 64 48
73 41 56
61 36 73
80 48 61
78 13 80
46 78 2
18 46 6
29 63 18
16 73 29
42 61 16
17 47 42
57 38 17
19 57 21
79 11 57
3 79 32
10 77 3
10 76 22
70 3 76
28 47 3
56 19 28
27 51 56
51 14 67
30 63 14
44 6 30
75 36 44
13 75 20
11 51 13
35 63 11
17 24 35
42 28 17
26 49 42
5 41 26
11 50 5
29 13 11
26 36 29
4 12 26
31 1 12
35 3 31
35 34 72
69 9 34
2 39 69
5 28 2
48 55 5
18 22 48
7 64 18
23 71 7
3 6 23
19 8 3
20 45 19
9 20 33
5 16 9
1 3 5
0
